### PR TITLE
[QC-491] PP Task to transform Quality Objects into TRFCollections

### PR DESCRIPTION
**Pull request by knopers8**
Needs https://github.com/AliceO2Group/AliceO2/pull/5554

### CODE REVIEW

File: Modules/Common/include/Common/TRFCollectionTask.h [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  void finalize(quality_control::postprocessing::Trigger, framework::ServiceRegistry&) override;
+
+ private:
+  quality_control::TimeRangeFlagCollection
+    transformQualities(quality_control::repository::DatabaseInterface& qcdb, const uint64_t timestampLimitStart, const uint64_t timestampLimitEnd);
+
+ private:
+  TRFCollectionTaskConfig mConfig;
+  uint64_t mLastTimestampLimitStart = 0;
+  //  o2::analysis::TimeRangeFlagCollection mTRFCollection;
```
Comment (Barthelemy) [inline Modules/Common/include/Common/TRFCollectionTask.h:53]: remove commented code.

File: Modules/Common/include/Common/TRFCollectionTask.h [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   TRFCollectionTask.h
+/// \author Piotr Konopka
+///
+
+#ifndef QUALITYCONTROL_SKELETONPOSTPROCESSING_H
```
Comment (Barthelemy) [inline Modules/Common/include/Common/TRFCollectionTask.h:16]: name incorrect

**Review by Barthelemy — CHANGES_REQUESTED** [review]
Very few comments, just cosmetics. Very good.

File: Framework/src/Quality.cxx [hunk 3]
```diff
@@ -18,12 +18,10 @@
 #include <iostream>
 #include <Common/Exceptions.h>
 
-ClassImp(o2::quality_control::core::Quality)
+ClassImp(o2::quality_control::core::Quality) //
 
-  // clang-format off
-namespace o2::quality_control::core
+  namespace o2::quality_control::core
```
[the remark this answers is not in the document]
Comment (knopers8) [reply Framework/src/Quality.cxx:23]: it is the ClassImp above which breaks the formatting by clang-format. But we can actually remove it, can't we?

[the remark this answers is not in the document]
Comment (Barthelemy) [reply Framework/src/Quality.cxx:23]: yes we can

[the remark this answers is not in the document]
Comment (Barthelemy) [reply Framework/src/Quality.cxx:23]: unimportant though, I merge

**Review by Barthelemy — APPROVED** [review]

File: Framework/src/Quality.cxx (hunk 3 above)
[the remark this answers is not in the document]
Comment (knopers8) [reply Framework/src/Quality.cxx:23]: I didn't push the other changes yet anyway, I will make another PR

**Pull request merged**